iron-fish/ironfish
# Added Ledger to Account Create, Import, Migration of Existing

## Summary
Added ledger flag to newly created accounts (create, import, multisig, DKG) and migrated db.

## Testing Plan
Ledger imports were not workingon my laptop so will need to confirm those next.
1. Create wallet
	1. false hard coded
	2. tested
2. Import wallet
	1. ledger 
		1. false hard coded
		2. tested
	3. ledger flag
		1. true hard coded
		2. tested
	4. import file
		1. wallet
			1. ledger value decoded
			2. tested
		2. ledger
			1. ledger value decoded
			2. tested
		3. long prompt
			1. ledger value decoded
3. Multisig
	1. particpants wallets
		1. false hard coded
		2. tested
4. DKG
	1. wallet
		1. false hard coded
		2. tested
	2. ledger
		1. true hard coded
		2. tested
5. Edge Cases
	1. Migration
		1. old wallets add false ledger flag
	2. Old wallet imports
		1. decoding added conditional false ledger property if it doesnt exist

## Documentation
N/A
```

## Breaking Change
Don't think so. Ledger flag is only being added to new and existing entries in db.
```
